src/components/Home/Home.js

```diff
@@ -1,24 +1,47 @@
-import { useTheme } from '@react-navigation/native';
-import React from 'react';
-import { Text, View } from 'react-native';
-import { useSelector } from 'react-redux';
+import React, { useEffect, useState } from 'react';
+import { View, Image, FlatList } from 'react-native';
+import useFetchMovies from '../../hooks/useFetchMovies';
+import Center from '../Center';
+import { getConfiguration } from '../../helpers/MoviesHelper';
+
+import logoIcon from 'assets/ic_logo/ic_logo.png';
 import styles from 'components/Home/styles';
-import TextStyles from 'helpers/TextStyles';
-import strings from 'localization';
-import { getUser } from 'selectors/UserSelectors';
+import MoviesListItem from 'components/MoviesListItem';
 
 function Home() {
-  const { colors } = useTheme();
-  const user = useSelector(getUser);
+  const [urlPrefix, setUrlPrefix] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [movies, fetchMore] = useFetchMovies();
+
+  useEffect(() => {
+    setIsLoading(true);
+    const fetchCfg = async () => {
+      const response = await getConfiguration();
+      setUrlPrefix(
+        response.images.base_url + response.images.backdrop_sizes[1]
+      );
+    };
+    fetchCfg();
+    setIsLoading(false);
+  }, []);
 
   return (
     <View style={styles.container}>
-      <Text style={[TextStyles.lightTitle, { color: colors.text }]}>
-        {strings.home}
-      </Text>
-      <Text style={{ color: colors.text }}>
-        {strings.homeMessage} {user?.name}
-      </Text>
+      <Center>
+        <Image source={logoIcon} style={styles.logoIcon} />
+      </Center>
+      {!isLoading && (
```

Comment:
What do you think about showing a loading indicator while it is loading?

Reply:
Totally! good catch!